src/Slider/Slider.tsx

```diff
@@ -36,18 +37,31 @@ export interface SliderProps {
     showTicks?: boolean;
     step?: number;
     theme?: SliderTheme;
+    appearance?: SliderAppearance;
```

Comment:
I don't like this approach, let's just follow the way we used to handle this which is through passing a boolean value of alternative which will be like this:
```tsx
<Slider value={value} alternative />
```
The reason is, there are no other appearances, there is just a default and another one. Comparing this to HTML elements, the disabled state is set similarly.

Reply:
I agree, the objective is to change the UI without having to change code or break it.
```typescript 
<Slider value={value} />
```
should work fine
If you really insist on keeping the appearance then please add a default value to it
```typescript
appearance?: SliderAppearance = SliderAppearance.normal;
```

Reply:
Omar I added the default value here https://github.com/sebgroup/react-components/pull/164/files#diff-3080399ac815fe720993abb6fa017bf1R64

Reply:
Yousif what about if we decide to add more appearances in the future? You approach only covers alternative and normal

Reply:
> I agree, the objective is to change the UI without having to change code or break it.
> 
> ```ts
> <Slider value={value} />
> ```
> 
> should work fine
> If you really insist on keeping the appearance then please add a default value to it
> 
> ```ts
> appearance?: SliderAppearance = SliderAppearance.normal;
> ```

The code won't break if you don't pass appearance. Yousif is just saying instead of appearance string we should just have alternative as boolean.

Reply:
There will be no more appearances, that's for sure. In fact, it's more likely that we will deprecate the alternative appearance eventually. We're trying to get people to abide by Design Library standards, not help renegades.